packages/sveltekit/src/client/load.ts

```diff
@@ -14,7 +14,11 @@ function sendErrorToSentry(e: unknown): unknown {
   const objectifiedErr = objectify(e);
 
   // We don't want to capture thrown `Redirect`s as these are not errors but expected behaviour
-  if (isRedirect(objectifiedErr)) {
+  // Neither 400 errors, given that they are not valuable.
+  if (
+    isRedirect(objectifiedErr) ||
+    (isHttpError(objectifiedErr) && objectifiedErr.status < 500 && objectifiedErr.status >= 400)
```

Comment:
so, the issue linked complains about 403 errors being captured, it talks about ignoring all 4xx errors, but we only ignore 400 - is that correct/on purpose?

Reply:
It's on purpose in the sense that we have the same logic on the server side and generally, afaik, we never want to capture 4xx errors by default.

Reply:
ahh wait, I got confused by the comment which says `neither 400` errors and thought this was filtering only 400 itself, not 403. all good then!

Reply:
ahh sorry, that was my mistake. 4xx is definitely the better description